core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -32,6 +32,7 @@ import org.apache.spark.executor._
 import org.apache.spark.scheduler._
 import org.apache.spark.storage._
 import org.apache.spark._
+import org.apache.hadoop.hdfs.web.JsonUtil
```

Comment:
Please file a JIRA, since that helps us to track and prioritize bugs.  Feel free to open a pull request if removing this import fixes your build.